helix-core/src/main/java/org/apache/helix/messaging/handling/HelixTaskExecutor.java

```diff
@@ -177,7 +177,7 @@ public HelixTaskExecutor(ParticipantStatusMonitor participantStatusMonitor,
     _lock = new Object();
     _statusUpdateUtil = new StatusUpdateUtil();
 
-    _timer = new Timer("HelixTaskExecutor_timer", true); // created as a daemon timer thread to handle task timeout
+    _timer = new Timer("HelixTaskExecutor_Timer", true); // created as a daemon timer thread to handle task timeout
```

Comment:
What's the difference of this line? Let's try to minimize the diff for reviewing.

Reply:
Capital "T" we can do a little better matching when finding leaked thread by name.